compiler/src/openxla/compiler/nvgpu/Dialect/CUDNN/Conversion/ConvertCUDNNToRuntime.cpp

```diff
@@ -196,6 +199,25 @@ func::FuncOp CudnnAPI::getConvolutionFunction(PatternRewriter &rewriter,
                  functionType);
 }
 
+func::FuncOp CudnnAPI::getReluFunction(PatternRewriter &rewriter,
+                                       ModuleOp module) {
+  MLIRContext *ctx = module->getContext();
+  auto tensor = CudnnTensorType::get(ctx);
+  auto f32 = rewriter.getF32Type();
+  auto i64 = rewriter.getI64Type();
+  auto i32 = rewriter.getI32Type();
+
+  SmallVector<Type> args = {/*input=*/tensor,   /*lower_clip=*/f32,
+                            /*upper_clip=*/f32, /*uid=*/i64,
+                            /*alignment=*/i64,  /*is_virtual=*/i32};
```

Comment:
`uid` and `alignment` are not explicit in the IR, they are generated at run time in the `CudnnModule`

Reply:
`CustomModule` Relu implementation was not updated to be consistent with other ops, I think uid and alignment were explicit in the original version, but now it's gone in every other op